app/src/RoomClient.js

```diff
@@ -1979,6 +1981,56 @@ export default class RoomClient
 		}
 	}
 
+	handleConsumerPreferredLayers()
+	{
+		let limit;
+
+		if (this._device.platform === 'desktop')
+		{
+			limit = window.config.qualityDecMaxN?window.config.qualityDecMaxN:3;
+		}
+		else
+		{
+			limit = window.config.qualityDecMobileMaxN?window.config.qualityDecMobileMaxN:1;
+		}
+
+		const activeVideoConsumers = new Array();
+
+		for (const consumer of this._consumers.values())
+		{
+			if (consumer.kind === 'video' && !consumer.paused && !consumer.closed &&
+				this._spotlights.peerInSpotlights(consumer.appData.peerId))
+			{
+				activeVideoConsumers.push(consumer);
+			}
+		}
+
+		activeVideoConsumers.forEach((consumer) =>
+		{
+			let spatialLayer = 100;
+
+			let temporalLayer = 100;
```

Comment:
Why 100 for both spacial and temporal layers?

Reply:
We need to figure out a better algorithm, I have been talking with @so010 about this. For now, it is either maximum quality (100) or lowest quality (0).